src/llmq/quorums_blockprocessor.cpp

```diff
@@ -103,7 +109,7 @@ void CQuorumBlockProcessor::ProcessMessage(CNode* pfrom, const std::string& strC
         LogPrintf("CQuorumBlockProcessor::%s -- received commitment for quorum %s:%d, validMembers=%d, signers=%d, peer=%d\n", __func__,
                   qc.quorumHash.ToString(), qc.llmqType, qc.CountValidMembers(), qc.CountSigners(), pfrom->id);
 
-        AddMinableCommitment(qc);
+        AddMinableCommitment(hash, qc);
```

Comment:
It's basically the only place this optimization is _really_ used and I'm not sure it's worth it, probably not a big deal to rehash it once. I'd prefer to keep it the way it is tbh.

Reply:
Hmm yeah, I might have been over-motivated with this one :) I'll remove it before merging (I see you already gave an utACK).

Reply:
Removed this from the involved commit and force-pushed.